package.json

```diff
@@ -13,6 +13,7 @@
     "client-sessions": "^0.8.0",
     "dotenv": "^5.0.1",
     "express": "^4.16.2",
+    "express-hbs": "^1.0.4",
```

Comment:
Do we still need standalone `hbs` below if we're including `express-hbs`?

Reply:
Probably not.